src/silx/gui/plot/PlotWindow.py

```diff
@@ -183,6 +183,12 @@ def __init__(
         self.yAxisLogarithmicAction.setVisible(logScale)
         self.addAction(self.yAxisLogarithmicAction)
 
+        self.yAxisArcsinhAction = self.group.addAction(
+            actions.control.YAxisArcsinhAction(self, parent=self)
+        )
+        self.yAxisArcsinhAction.setVisible(logScale)
+        self.addAction(self.yAxisArcsinhAction)
```

Comment:
Adding a new axis scaling changes its control from a boolean to an enum (it should have been so since the beginning).
It would make sense to me to replace `yAxisLogarithmicAction` with a toolbutton with a drop-down.